README.md

```diff
@@ -20,6 +20,13 @@ Supported platforms:
 
 ## Usage
 
-This repository contains C-language examples that link to the native library targets and perform basic rendering and computation. You can test them with `make`.
+This repository contains C-language examples that link to the native library targets and perform basic rendering and computation.
+To run the C triangle example, install a C compiler, then run these commands at the root of the repo:
+```
+rustup toolchain install nightly
+make examples-native
```

Comment:
I think this would fail to run `cbindgen` if not installed

Reply:
Maybe we should separate header generation out of this target

Reply:
It doesn't seem to complain at all without cbindgen installed.

Reply:
Turns out it is needed, so I added it.
I see how I got confused though.